### ISSUE TITLE
Installation error on NVIDIA GeForce RTX 4090

### ISSUE THREAD

**Issue by bilalaliofficial** [collabora/WhisperFusion#61]
I’m trying to install this package on an RTX 4090 GPU, but it is giving me the following error:

`./build-whisper.sh: line 17:    12 Aborted                 (core dumped) python3 build.py --output_dir whisper_small_en --use_gpt_attention_plugin --use_gemm_plugin --use_bert_attention_plugin --enable_context_fmha --model_name small.en
whisperfusion-1 exited with code 134
`
![image](https://github.com/user-attachments/assets/01538c95-b96b-4449-a8bb-288216d22a6d)

 Nvidia SMI

![image](https://github.com/user-attachments/assets/2131339f-bfaf-4a1a-a6bf-878ebe241ba9)

Please help me to resolve this issue

**Comment by makaveli10** [comment]
Isn't the gpu is fully utilised already, can you try again after clearing some space on the GPU?

**Comment by bilalaliofficial** [comment]
Yes, I got it and solved this issue and it's showing no error and running but when I'm trying to load on browser not working

![image](https://github.com/user-attachments/assets/c2a8ff6a-8def-4fa9-bb5f-f08c3f9e73c8)
![image](https://github.com/user-attachments/assets/48dc27ff-c70e-4a62-bf91-9eed4f69654c)

![image](https://github.com/user-attachments/assets/522ce5be-79d9-4e59-9f61-c1ef4456f97e)

**Comment by makaveli10** [comment]
Its should be available on `localhost:8000`

**Comment by bilalaliofficial** [comment]
Yes, I got it but this GPU is deployed on the network, and I'm accessing it from my laptop.

**Comment by makaveli10** [comment]
In that case can you try forwarding the ports(`8000, 8888, 6006`) from the network to your local and then try accessing `localhost:8000`

**Comment by goranskular** [comment]
How did you resolve the core dump issue? Thanks

**Comment by ananpalliyal** [comment]
@bilalaliofficial hello did you get it running? I'm also trying to run whisperfusion on a rtx 4090 cloud instance the problem is that this gpu instance doesn't support docker, so I have to do everything manually.
also, where did you get your rtx 4090 from?
